ecl/hql/reservedwords.cpp

```diff
@@ -176,7 +181,7 @@ static const char * eclReserved7[] = { //Comparison ops
     "<",
     "<=",
     ">",
-    //">=",     //x.<y>= n  really wants to be processed as x.<y> = n, not x.<y >=
```

Comment:
Ok, I think the best thing to do is leave it commented out and then I'll add it to the internals so this whole thing actually works. The reason it was commented out in the first place was so that if used for syntax highlighting it wouldn't be confusing.